optimum/intel/openvino/utils.py

```diff
@@ -145,3 +145,9 @@ def _print_compiled_model_properties(compiled_model):
                 logger.info(f"  {k}: {value}")
         except Exception:
             logger.error(f"[error] Get property of '{k}' failed")
+    try:
+        logger.info("EXECUTION_DEVICES:")
+        for device in compiled_model.get_property("EXECUTION_DEVICES"):
+            logger.info(f"  {device}: {Core().get_property(device, 'FULL_DEVICE_NAME')}")
+    except Exception:
+        logger.error("[error] Get FULL_DEVICE_NAME failed")
```

Comment:
in which circonstance would this fail ?

Reply:
> in which circonstance would this fail ?

If there is a bug (or API change) in OpenVINO. I did the warning like this to be consistent with the other warning when getting a property fails. I'm also fine with setting this to pass, I just don't want any exceptions from this function to propagate.